app/src/main/java/com/example/androidthings/assistant/AssistantActivity.java

```diff
@@ -173,12 +180,23 @@ public void onSpeechRecognition(List<SpeechRecognitionResult> results) {
                     }
                 })
                 .setConversationCallback(new ConversationCallback() {
+                    @Override
+                    public void onResponseStarted() {
+                        if (mDac != null) {
+                            try {
+                                mDac.setSdMode(Max98357A.SD_MODE_LEFT);
+                                mLed.setValue(true);
+                            } catch (IOException e) {
+                                Log.e(TAG, "error enabling DAC", e);
+                            }
+                        }
+                    }
+
                     @Override
                     public void onAudioSample(ByteBuffer audioSample) {
                         if (mLed != null) {
                             try {
-                                mDac.setSdMode(Max98357A.SD_MODE_SHUTDOWN);
-                                mLed.setValue(false);
+                                mLed.setValue(!mLed.getValue());
```

Comment:
I think @Fleker removed this behavior on purpose because it was introducing flakiness in the playback.

Reply:
Yes. I think toggling the LED may introduce some performance issues.